.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "tesseract"]
```

Comment:
Can you add the [submodules option](https://github.com/actions/checkout#:~:text=%23%20Default%3A%20false-,submodules%3A%20%27%27,-%23%20Add%20repository) to the github workflow?

```diff
diff --git a/.github/workflows/rust.yml b/.github/workflows/rust.yml
index 34670ed..b21b157 100644
--- a/.github/workflows/rust.yml
+++ b/.github/workflows/rust.yml
@@ -18,6 +18,8 @@ jobs:
     - name: Install Leptonica
       run: sudo apt-get install libleptonica-dev clang
     - uses: actions/checkout@v2
+      with:
+        submodules: true
     - name: Build
       run: cargo build --verbose
     - name: Run tests
```